Lib/test/test_codecs.py

```diff
@@ -1753,6 +1754,14 @@ def test_file_closes_if_lookup_error_raised(self):
 
             file().close.assert_called()
 
+    def test_copy(self):
+        orig = codecs.lookup('utf-8')
+        dup = copy.copy(orig)
+
+        self.assertEqual(dup.encode, orig.encode)
+        self.assertEqual(dup.name, orig.name)
+        self.assertEqual(dup.incrementalencoder, orig.incrementalencoder)
```

Comment:
Please add also test for `_is_text_encoding`. And since it is True by default, it is better to use the codec in which it is set to False instead of utf-8.